templates/markdownx/widget.html

```diff
@@ -0,0 +1,8 @@
+<div class="markdownx row">
```

Comment:
If we drop bootstrap dep we could use something like this [SO post](http://stackoverflow.com/a/17217959) to handle side-by-side display. If that doesn't work then lets just show the preview below the `textarea`.